frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -24,7 +24,10 @@ import {
   k8sUpdate,
   K8sVerb,
 } from '@console/internal/module/k8s';
-import { ServiceModel as KnServiceModel } from '@console/knative-plugin';
+import {
+  ServiceModel as KnServiceModel,
+  ServiceModel as knSvcModel,
```

Comment:
You importing the same model twice. Remove the duplicates.
```suggestion
  ServiceModel as KnServiceModel,
```